### PR TITLE
Add tutorial samples for PAI

### PR DESCRIPTION
**Pull request by TobeyQin**
Add tutorial samples for PAI.
1. Frameworks: TensorFlow, PyTorch, MxNet, keras
2. Samples can both run on Visual Studio and Command Line.
3. Include Json file, Support submit to PAI through Visual Studio Tools for AI or submit job to PAI portal directly.
4. Add README file for each Visual Studio Solution (or each framework).

### CODE REVIEW

File: examples/tutorial/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    ```
+
+## Runing samples on Visual Studio
+
+    - Set the "Startup File".
+
+    - Right click the startup Python script, and click "Start without Debugging" or "Start with Debugging" context menus.
+
+
+## Submit samples to Microsoft PAI through Visual Studio Tools for AI
```
Comment (YanjieGao) [inline examples/tutorial/README.md]: Could these examples support submit to PAI by pai's webportal ?

Comment (YanjieGao) [reply examples/tutorial/README.md]: How about tell user a case how to submit example by webportal?

Comment (TobeyQin) [reply examples/tutorial/README.md]: I usually submit job with Tools for AI, it is easy to upload code and data. I can have a try to submit from webportal. Or would you give a tutorial document about that?

File: examples/tutorial/README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+- CNTK (both BrainScript and Python languages)
+- Tensorflow
+- PyTorch
+- Caffe2
+- Keras
+- MXNet
+- Chainer
+- Theano
+
+**NOTICE: ** If you submit your job through OPenPAI portal directly, you don't need Visual Studio solution and project files.
```
Comment (sterowang) [inline examples/tutorial/README.md]: >OPenPAI [](start = 44, length = 7)

Wrong upper case of P? Should be "OpenPAI"?

File: examples/tutorial/caffe2/MNIST/caffe2_mnist.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "jobName": "caffe2_mnist",
+  "image": "openpai/pai.example.caffe2",
+  "codeDir": "$PAI_DEFAULT_FS_URI/tutorial/caffe2_mnist/code",
+  "authFile": "",
```
[the remark this answers is not in the document]
Comment (TobeyQin) [reply examples/tutorial/caffe2/MNIST/caffe2_mnist.json]: OK. I will remove it.

**Comment by TobeyQin** [comment]
I will close this PR because we finally decided to keep these samples in [Samples for AI](https://github.com/Microsoft/samples-for-ai) repo and mentioned PAI submission procedure. Thank you very much for your review. @YanjieGao @qyyy

**Pull request closed without merging**